dts/bindings/led/led-controller.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2020 Seagate Technology LLC
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for LED controllers and child LEDs
+
+properties:
+    label:
+      required: true
+    num-leds:
```

Comment:
No it is not necessary. I have been hesitating about removing it. For simple controllers this allows to skip the LED subnode declarations and/or the counting in the driver. I am not sure it is suitable but it may be convenient. So I kept it as an optional property.

Reply:
We can always add properties later; perhaps we can leave it out until it is really needed?

Reply:
Yes. I'll remove it.